test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -41,22 +41,32 @@
 
 @tags('multisched')
 class TestSchedulerInterface(TestInterfaces):
+
     """
     Test suite to test different scheduler interfaces
     """
 
     def setUp(self):
         # delete all schedulers except default
-        self.server.manager(MGR_CMD_DELETE, SCHED, id="@default")
+        self.server.manager(MGR_CMD_DELETE,
+                            SCHED,
+                            id="@default")
```

Comment:
I just saw your response to my last comment.  Please ignore this one.  I am fine with the delay to PTL infrastructure.